### PR TITLE
Full website design

### PR DESCRIPTION
**Pull request by ridaarif98**
- Create a website consisting of 3 pages: `Home`, `Calculator`, and `Quote`.
- The general layout  matches with wireframes
- Add styles to improve the look and feel. 
- For every page, create a route.
- Navigation links for all 3 routes are displayed on each page
- Use `React Router` to make the routes work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,11 @@
    - Make simple calculations.
    - Read a random math-related quote.
 
+
+## Live
+
+  [LiveDemo](https://ridaarif98.github.io/)
```
Comment (dasileker) [inline README.md]: - [x] Please fix the live demo link, it's not working

File: src/components/Calculator.js [hunk 2]
```diff
@@ -2,7 +2,11 @@ import React, { useState } from 'react';
 import calculate from '../logic/calculator';
 
 function CalculatorDesign() {
-  const [inputs, setInput] = useState({ total: 0, next: null, operation: null });
+  const [inputs, setInput] = useState({
+    total: 0,
+    next: null,
+    operation: null,
```
Comment (dasileker) [inline src/components/Calculator.js:8]: - [x] Please fix your calculator, I can't do any operation. please check your **operation.js** and fix your logic.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @ridaarif98,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- You have done a great job on design, but you have issue in your logic **operate.js** I can't do any calculation in the calculator fix it please.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @ridaarif98  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**